src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.awt.List;
+import java.util.*;
```

Comment:
import specific imports rather than everything. can use eclipse's 'organise imports' to do this